apps/taiga-lumbermill/src/dashboards/crypto/components/swap/swap.component.html

```diff
@@ -0,0 +1,92 @@
+@let info = info$ | async;
+@if (info) {
+    @let data = info.data;
+    <div
+        tuiAppearance="whiteblock"
+        tuiCardLarge="normal"
+        [style.height]="'100%'"
+    >
+        <header tuiHeader>
+            <h2
+                tuiTitle
+                [style.text-align]="'center'"
+            >
+                Swap
+            </h2>
+        </header>
+        @for (type of swapService; track $index) {
+            <div>
+                <div tuiCell="l">
+                    <div TuiTitle>{{ type.title }}</div>
+                    <div TuiTitle>Balance: 1000 {{ swapService[$index].chosen.toUpperCase() }}</div>
+                </div>
+                <button
+                    tuiButton
+                    tuiChevron
+                    type="button"
+                    class="choose-crypto"
+                    [tuiDropdown]="dropdownContent"
+                    [tuiDropdownManual]="swapService[$index].status"
```

Comment:
dropdown should close after selection and after click outside the list. Try to use DropdownOpen (first example) https://taiga-ui.dev/next/directives/dropdown-open